internal/rbd/rbd_util.go

```diff
@@ -963,7 +963,7 @@ func (rv *rbdVolume) createSnapshot(ctx context.Context, pOpts *rbdSnapshot) err
 }
 
 func (rv *rbdVolume) deleteSnapshot(ctx context.Context, pOpts *rbdSnapshot) error {
-	util.DebugLog(ctx, "rbd: snap rm %s using mon %s", pOpts, pOpts.Monitors)
+	util.DebugLog(ctx, "rbd: snap rm %s@%s using mon %s", rv, pOpts.String(), pOpts.Monitors)
```

Comment:
this will look weird. `pOpts.String()` should contain the full rbd image path, including `@` and what comes before it